statsd.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+var TXStatsD = require('node-txstatsd');
+
+// StatsD wrapper
+function StatsD(statsdHost, statsdPort) {
+    this.statsd = new TXStatsD({
+        host: statsdHost,
+        port: statsdPort,
+        prefix: 'restbase.routes.',
```

Comment:
Yup. That's what the name in the config is (partly) for. Just haven't done that yet.